content/en/smart-contracts/fundamentals/erc-20-quickstart/index.md

```diff
@@ -213,6 +213,34 @@ Now that we've successfully compiled our contract, we need to deploy it somewher
 
 On the Filecoin network, a new set of blocks, also called a tipset, is created every thirty seconds. When deploying a contract, the transaction needs to be received by the network, and then the network needs to confirm the contract. This process takes around one to two tipsets to process -- or around 60 to 90 seconds.
 
+## Verify your contract (main net deployment only)
```

Comment:
Note: Contract verification should be available on testnets too.

```suggestion
## Verify your contract (mainnet deployment only)
```

Reply:
is it important to verify contracts on testnet though?

Reply:
We should indicate it's an option too. Devs can test the verifiability early so they can be onto problems (theirs or the explorers's) to minimise surprise factors on mainnet.